### PR TITLE
EDU-1557 Support document category creation

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1557

### CODE REVIEW

File: src/components/maintenance/document-category-metadata-modal.yml [hunk 1]
```diff
@@ -5,3 +5,12 @@ documentCategoryMetadataModal:
   modalCreateTitle:
     en: New document category
     de: Neue Dokument-Kategorie
+  nameRequired:
+    en: Please specify a document category name
+    de: Bitte gib einen Dokument-Kategorien-Namen an
```
Comment (ahelmberger) [inline src/components/maintenance/document-category-metadata-modal.yml:10]: Kategorie~~n~~

File: src/components/maintenance/document-category-metadata-modal.yml [hunk 2]
```diff
@@ -5,3 +5,12 @@ documentCategoryMetadataModal:
...
+  nameIsInUse:
+    en: The category name is already used
+    de: Die Kategoriename ist bereits verwendet
```
Comment (ahelmberger) [inline src/components/maintenance/document-category-metadata-modal.yml:13]: **Der** Kategoriename...

**Comment by ahelmberger** [comment]
Works nicely :+1:

**Review by ahelmberger — APPROVED** [review]

File: src/components/maintenance/document-category-metadata-modal.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/maintenance/document-category-metadata-modal.yml]: Corrected, thanks!

**Pull request merged**